tests/test_deploy.py

```diff
@@ -324,3 +324,22 @@ def test_rsync_command_delete_no(tmpdir, mocker, env):
             "example.com:/",
         ],
     )
+
+
+def test_rsync_local(tmpdir, mocker, env):
```

Comment:
These tests seem to be quite repetitive. I'll probably refactor them to use `pytest.mark.parametrize` in a follow-up.